ts/RootContainer.tsx

```diff
@@ -145,8 +146,12 @@ class RootContainer extends React.PureComponent<Props> {
         {Platform.OS === "android" && <FlagSecureComponent />}
         <Navigation />
         {shouldDisplayVersionInfoOverlay && <VersionInfoOverlay />}
-        {bpdTestOverlay && <BpdTestOverlay />}
-        {cgnTestOverlay && <CgnTestOverlay />}
+        {cgnTestOverlay && (
+          <BetaTestingOverlay title="🛠️ CGN TEST VERSION 🛠️" />
+        )}
+        {bpdTestOverlay && (
+          <BetaTestingOverlay title="🛠️ BPD TEST VERSION 🛠️" />
```

Comment:
With this implementation BPD Overlay would be missing of some of the informations previously shown see: https://github.com/pagopa/io-app/blob/master/ts/features/bonus/bpd/components/BpdTestOverlay.tsx#L66